src/schema/nodes/equation.ts

```diff
@@ -17,64 +17,63 @@
 import { ObjectTypes } from '@manuscripts/json-schema'
 import { NodeSpec } from 'prosemirror-model'
 
+import { convertTeXToMathML } from '../../mathjax/tex-to-mathml'
 import { ManuscriptNode } from '../types'
 
 interface Attrs {
   id: string
-  MathMLStringRepresentation: string
-  SVGStringRepresentation: string
-  TeXRepresentation: string
+  content: string
 }
 
 export interface EquationNode extends ManuscriptNode {
   attrs: Attrs
 }
 
+export const getEquationContent = (p: string | HTMLElement) => {
+  const element = p as HTMLElement
+  const container = element.querySelector('alternatives') ?? element
+  let content: string | null = ''
+  for (const child of container.childNodes) {
+    // remove namespace prefix
+    // TODO: real namespaces
+    const nodeName = child.nodeName.replace(/^[a-z]:/, '')
+
+    switch (nodeName) {
+      case 'tex-math':
+        content = convertTeXToMathML((child as Element).outerHTML, true)
```

Comment:
Thinking about it more, I feel that we shouldn't do this conversion. In transform we should keep the data as is, and the FE can handle the different cases.